docs/docs/reference/built-in-components/gatsby-plugin-image.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Gatsby Image plugin
+---
+
+This guide will show you how to configure your images, including choosing layouts, placeholders and image processing options. If you would like to learn how to set up the image plugins and use images on your site, see [the how-to guide](/docs/how-to/images-and-media/using-gatsby-plugin-image).
+
+## Components
+
+The Gatsby Image plugin includes two components to display responsive images on your site, used for static and dynamic images.
+
+- **`StaticImage`:** Use this if the image is the same every time the component is used. _Examples: site logo, index page hero image_
+- **`GatsbyImage`:** Use this if the image is passed-in to the component as a prop. _Examples: Blog post hero image, author avatar_
+
+## Image options
```

Comment:
Are these options all optional? (Sounds silly when I write it out like that...)

It might be good to add a quick intro sentence to give this section some context. I don't love this wording, but something like: "If you'd like to override the default behavior of `StaticImage` and `GatsbyImage`, you can pass in one or more additional options."

Reply:
Nothing is required except for alt. Which is not explicitly listed but in all the examples, so we should probably adjust that.